contracts/TellorProvider.sol

```diff
@@ -0,0 +1,53 @@
+pragma solidity 0.4.24;
+
+import "./ITellorGetters.sol";
+import "./IMedianOracle.sol";
+
+contract TellorProvider{
+
+    ITellorGetters public tellor;
+    IMedianOracle public medianOracle;
+
+    
+    struct TellorTimes{
+        uint128 time0;
+        uint128 time1;
+    }
+    TellorTimes public tellorReport;
+    uint256 constant TellorID = 10;
+
+
+    constructor(address _tellor, address _medianOracle) public {
+        tellor = ITellorGetters(_tellor);
+        medianOracle = IMedianOracle(_medianOracle);
+    }
+
+    function pushTellor() external {
+        (bool retrieved, uint256 value, uint256 _time) = getTellorData(); 
+        //Saving _time in a storage value to quickly verify disputes later
```

Comment:
Hey am I missing something or should the report only get pushed if `retrieved` is true?

Reply:
`retrieved` is true if the value count is greater than 0. `TellorID` 10 has months of data with a value count of over 4500, so `retrieved` will always be true when using the current Tellor contracts.